### DISCUSSION TITLE
Issue with showFilePickerOpts parameters

### DISCUSSION THREAD

**Discussion by AnaChips** [mdn/mdn-community#289] [Content]
Hello everyone, facing the following issues when using window.showFilePickerOpts, as given in example
1. The array of MIME type extensions does not filter files: With the following line of code: ''image/*': ['.png', '.gif', '.jpeg', '.jpg']' I expected to be able to choose files only with the extensions mentioned i.e. png, gif and jpeg. On the contrary, I was able to see image files of all formats like svg, tiff as well.  
2. The 'multiple' flag is not working :  With 'multiple: true' , I expected to be able to select multiple files, however I am unable to figure out how they are returned, as the getFile() method returns only a single file.
![Untitled](https://user-images.githubusercontent.com/25129720/203022632-cc41254f-46b8-4000-a6c8-18ac2e6074ca.png)

**Comment by Rumyra** [comment]
Thanks @AnaChips - can you tell us what browser (and what version specifically) and what platform you are using - thanks 👍
